pilot/pkg/serviceregistry/kube/controller/ambient/ambientindex.go

```diff
@@ -202,7 +207,7 @@ func New(options Options) Index {
 		WorkloadServices,
 		WorkloadEntries,
 		ServiceEntries,
-		AllPolicies,
```

Comment:
Why is AllPolicies removed

Reply:
This was an input variable that was not used already